docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+services:
+  app:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8000:8000"
+    volumes:
+      - ./:/app
```

Comment:
It is better not to mount the whole directory as a volume in the final version, it should be done for development purposes only